ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -760,6 +760,63 @@ def test_chainexpiration_check_without_cert(self, move_ipa_ca_crt):
             assert check["kw"]["error"] == error_text
             assert check["kw"]["msg"] == msg_text
 
+    @pytest.fixture
+    def modify_cert_trust_attr(self):
+        """
+        Fixture to modify trust attribute for Server-cert and
+        revert the change.
+        """
+        self.master.run_command(
+            [
+                "certutil",
+                "-M",
+                "-d",
+                paths.PKI_TOMCAT_ALIAS_DIR,
+                "-n",
+                "Server-Cert cert-pki-ca",
+                "-t",
+                "CTu,u,u",
+                "-f",
+                paths.PKI_TOMCAT_ALIAS_PWDFILE_TXT,
```

Comment:
Black I assume? It is honestly more readable for me to keep option/value pairs together, e.g.

```
"-M",
"-d", paths.PKI_TOMCAT_ALIAS_DIR,
"-n", "Server-Cert cert-pki-ca",
...
```
I also wonder if this fixture would benefit to take the host, db, nickname and trust as options so we could modify multiple values at once. Not a deal breaker, this is the minimum needed to test the check.

Reply:
@rcritten yes i had used black, have made the changes to keep option/value pair together.